src/lib/y2partitioner/widgets/fstab_options.rb

```diff
@@ -60,29 +77,40 @@ def ui_term_with_vspace(widget)
       end
 
       def delete_fstab_option!(option)
-        # The options can only be modified using BlkDevice#fstab_options=
-        filesystem.fstab_options = filesystem.fstab_options.reject { |o| o =~ option }
+        # The options can only be modified using MountPoint#mount_options=
+        mount_point.mount_options = mount_point.mount_options.reject { |o| o =~ option }
       end
 
       def add_fstab_options(*options)
-        # The options can only be modified using BlkDevice#fstab_options=
-        # so we can't just append options with << or push
-        new_options = filesystem.fstab_options
-        options.each { |opt| new_options << opt }
-        filesystem.fstab_options = new_options
+        # The options can only be modified using MountPoint#mount_options=
+        mount_point.mount_options = mount_point.mount_options + options
       end
 
       alias_method :add_fstab_option, :add_fstab_options
 
     private
 
+      # Currently editing filesystem
+      #
+      # @return [Y2Storage::Filesystems::Base]
+      def filesystem
+        @controller.filesystem
+      end
+
+      # Mount point of the current filesystem
+      #
+      # @return [Y2Storage::MountPoint]
+      def mount_point
+        filesystem.mount_point
+      end
+
       # Common regexp checkbox widgets init.
       def init_regexp
-        i = filesystem.fstab_options.index { |o| o =~ self.class::REGEXP }
+        i = mount_point.mount_options.index { |o| o =~ self.class::REGEXP }
 
         self.value =
           if i
-            filesystem.fstab_options[i].gsub(self.class::REGEXP, "")
+            mount_point.mount_options[i].gsub(self.class::REGEXP, "")
```

Comment:
:thumbsup: for bringing more consistency into that naming that has somewhat gone out of hand (mount_options vs. fstab_options)